console/src/components/Common/Headers/utils.js

```diff
@@ -0,0 +1,50 @@
+const emptyHeader = {
+  name: '',
+  value: '',
+  type: 'static',
+};
+
+export const transformHeaders = (headers = []) => {
+  return headers
+    .map(h => {
+      const transformedHeader = {
+        name: h.name,
+      };
+      if (h.type === 'static') {
+        transformedHeader.value = h.value;
+      } else {
+        transformedHeader.value_from_env = h.value;
+      }
+      return transformedHeader;
+    })
+    .filter(h => !!h.name && (!!h.value || !!h.value_from_env));
```

Comment:
It would. But I have been adding `!!` recently to make the code feel semantically correct. Adding `!!` makes it explicit that the value is being used as boolean.